src/utils/vscode-utils.ts

```diff
@@ -37,7 +43,7 @@ export class VSCodeService {
             return;
         }
 
-        var files = fs.readdirSync(startPath).filter(e => e.indexOf('node_modules'))
+        var files = fs.readdirSync(startPath).filter(e => e.indexOf('node_modules'));
         for (var i = 0; i < files.length; i++) {
             var filename = path.join(startPath, files[i]);
             var stat = fs.lstatSync(filename);
```

Comment:
Use `let`/`const` instead of `var`.

Reply:
Hm, thanks I thought the linters should handle this one..